lib/SPIRV/OCLToSPIRV.cpp

```diff
@@ -168,9 +168,10 @@ bool OCLToSPIRVBase::runOCLToSPIRV(Module &Module) {
   auto Src = getSPIRVSource(&Module);
   // This is a pre-processing pass, which transform LLVM IR module to a more
   // suitable form for the SPIR-V translation: it is specifically designed to
-  // handle OpenCL C built-in functions and shouldn't be launched for other
+  // handle OpenCL C/CPP built-in functions and shouldn't be launched for other
```

Comment:
Out of curiosity, what is the reason for only running this transform for these types of sources?

Reply:
These are the only OpenCL related sources and this particular module deals with OpenCL sources.